### DISCUSSION TITLE
minio cluster cpu load imbalance

### DISCUSSION THREAD

**Discussion by Gordon-Gan-Jiang** [minio/minio#18671] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior
The CPU load conditions of the four nodes in the cluster should be similar.

## Current Behavior

The minio cluster has four nodes, and the CPU usage of two nodes remains above 100%，
![image](https://github.com/minio/minio/assets/62733672/ef4e6414-9e94-443c-ad7f-d86ab72a623f)

## Possible Solution

Through monitoring, it was found that the two nodes with higher CPUs kept scanning objects; tried the command "mc admin config set minio scanner delay", but still the problem could not be solved.
![image](https://github.com/minio/minio/assets/62733672/729707b6-c178-41ec-9841-455f1ef111d3)

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`):mc version RELEASE.2023-12-14T00-37-41Z
* Server setup and configuration: The number of cores per node is 16 cores
![image](https://github.com/minio/minio/assets/62733672/a62d0a8c-a737-432f-b994-62061ed6a938)
Supplementary explanation: Each disk size is 8TB, and disk usage takes up 74% of the total capacity.
* Operating System and version (`uname -a`): Linux version 5.4.0-135-generic

**Comment by klauspost** [comment, marked as answer]
We do not do performance analysis as a community service. Since you are clearly running a non-hobby setup, refer to our [subscriptions](https://min.io/pricing) for support options.
